tests/unittests/CoreGraphics/CGContextTests.mm

```diff
@@ -442,7 +442,7 @@ virtual void SetUp() {
     // Check the canvas context pixel after drawing
     dataPtr = static_cast<BYTE*>(CGBitmapContextGetData(context.get()));
     ASSERT_NE(dataPtr, nullptr);
-    EXPECT_EQ(dataPtr[0], 0x97);
+    EXPECT_EQ(dataPtr[0], 0x98);
 }
```

Comment:
I get that, but what does 0x98 mean? I understand that we're checking the first pixel to be a particular something but that something is pretty ambiguous just from looking at the code.

---
In reply to: [93538970](https://github.com/Microsoft/WinObjC/pull/1557#discussion_r93538970) [](ancestors = 93538970,93538810)